roofit/roofitcore/inc/RooExtendedTerm.h

```diff
@@ -31,11 +31,8 @@ class RooExtendedTerm : public RooAbsPdf {
   Double_t evaluate() const { return 1. ; }
 
   virtual ExtendMode extendMode() const { return CanBeExtended ; }
+  // Return number of expected events, i.e. the value of the associated n parameter
```

Comment:
`///` and maybe check if the `i.e.` trips doxygen into stopping the sentence there. It should be OK, since I have never seen that problem, but it doesn't hurt to check it.
Maybe also quote `n` or explain it a bit better (unless it's clear from the \class description).